site/content/docs/5.0/migration.md

```diff
@@ -7,6 +7,12 @@ aliases: "/migration/"
 toc: true
 ---
 
+## v5.0.0-beta3
+
+### JavaScript
+
+- All plugins are now allowed to accept a CSS selector as the first argument. You can either pass a DOM element or any valid CSS selector to create a new instance of the component.
```

Comment:
Maybe we should add an example here?

And perhaps in the JavaScript page https://getbootstrap.com/docs/5.0/getting-started/javascript/ instead of adding examples in each component page?

Reply:
Done :+1: 
https://deploy-preview-32245--twbs-bootstrap.netlify.app/docs/5.0/getting-started/javascript/#css-selectors-in-constructors